builder/azure/arm/step_delete_resource_group.go

```diff
@@ -95,7 +95,7 @@ func (s *StepDeleteResourceGroup) deleteDeploymentResources(ctx context.Context,
 			resourceType,
 			resourceName))
 
-		err := retry.Config{
+		_ = retry.Config{
```

Comment:
```suggestion
		retry.Config{
```

Reply:
as well as `return nil` on line 109. Can't figure out how to suggest that on GitHub... 😉

Reply:
@paulmey @azr Done.